packages/opentelemetry-resources/src/platform/node/detectors/ProcessDetector.ts

```diff
@@ -32,6 +32,8 @@ class ProcessDetector implements Detector {
       [SemanticResourceAttributes.PROCESS_COMMAND_LINE]:
         process.argv.join(' ') || '',
       [SemanticResourceAttributes.PROCESS_RUNTIME_VERSION]: process.versions.node,
+      [SemanticResourceAttributes.PROCESS_RUNTIME_NAME]: 'nodejs',
```

Comment:
Could you also add them for the browser platform ?

Reply:
added in the latest commit, please check.